stdlib/public/core/ObjectIdentifier.swift

```diff
@@ -59,7 +59,7 @@ public struct ObjectIdentifier: Sendable {
 
   /// Creates an instance that uniquely identifies the given metatype.
   ///
-  /// - Parameter: A metatype.
+  /// - x: A metatype.
```

Comment:
```suggestion
  /// - Parameter x: A metatype.
```

Reply:
Why should this parameter use a different syntax in the doc comment?  Both `- Parameter x:` and `- x:` are understood as the same thing when processed by DocC.  The rest of the standard library uses the plain dash-colon syntax.  I'd prefer to be consistent here.

Reply:
> Both `- Parameter x:` and `- x:` are understood as the same thing when processed by DocC.

As far as I know, `- x:` on its own would become part of the "Discussion" section, rather than the "Parameters" section. I think the two options are:

1.  As suggested, change the list item to be recognized as a parameter. This is consistent with the preceding documentation of the same file.

https://github.com/swiftlang/swift/blob/043a9fd15c75e364de91d5f81952801230b863c1/stdlib/public/core/ObjectIdentifier.swift#L53-L58

2.  Nest the list item in a Parameters list. Perhaps this is what you are referring to?

```suggestion
  /// - Parameters:
  ///   - x: A metatype.
```

Reply:
Thanks for catching this @benrimmington — I didn't notice there was no `- Parameters:` directive above, so my change turned one incorrect markup into a different still-incorrect one.